test/NODE_ENV.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var vm = require('vm');
+
+var eq = require('./internal/eq');
+var throws = require('./internal/throws');
+
+
+suite('NODE_ENV', function() {
+
+  var source = fs.readFileSync(path.join(__dirname, '..', 'index.js'), 'utf8');
```

Comment:
It looks like this breaks browser tests.

Reply:
Indeed. I excluded the file in this context, which seems reasonable.